recipes/libraw/all/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(LibRaw)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+file(GLOB_RECURSE headers "source_subfolder/libraw/*.h")
+
+set(sources
+    "source_subfolder/src/libraw_c_api.cpp"
+    "source_subfolder/src/libraw_cxx.cpp"
+    "source_subfolder/src/libraw_datastream.cpp"
+    "source_subfolder/internal/dcraw_common.cpp"
+    "source_subfolder/internal/dcraw_fileio.cpp"
+    "source_subfolder/internal/demosaic_packs.cpp"
+)
+
+if(WIN32)
+    add_definitions(-DWIN32)
+
+    if(BUILD_SHARED_LIBS)
+        add_definitions(-DLIBRAW_BUILDLIB)
+    else()
+        add_definitions(-DLIBRAW_NODLL)
+    endif()
+endif()
+
+add_definitions(-DUSE_JPEG -DUSE_JPEG8)
```

Comment:
I would prefer `target_compile_definitions` because is not global. But I'm fine with this.